frontend/packages/dev-console/src/components/import/image-search/ImageSearch.scss

```
@@ -0,0 +1,3 @@
+.odc-imagesearch-helptextinvalid {
```

```suggestion
.odc-image-search__help-text-invalid {
```

You don't lose points for using more characters 😜